docs/DeveloperGuide.md

```diff
@@ -137,55 +137,131 @@ Classes used by multiple components are in the `seedu.clinic.commons` package.
-If two entries of `ct/TYPE` are found, the latter entry will be used as the argument.
-If the deletion command asks for deletion of product (indicated by the `TYPE` keywords `ps` and `pw`), the additional check
-for prefix `pd/` and valid product name will also be conducted.
-The code will throw a **ParseException** if the check fails.
-Afterwards, all the valid arguments will form a `DeleteCommand`, which will be executed.
-
-Step 2. The `DeleteCommand` is executed via a call from `LogicManager`. The execution is first classified into Supplier-related deletion
-and Warehouse-related deletions via the `targetType` attribute. Under each category, The execution splits into deletion of an entire supplier/warehouse
-or its particular product.
-
-Step 3. The method will retrieve the displayed list of warehouse/supplier via `model#getFilteredWarehouseList()`.
-It locates the warehouse/supplier user wants to delete (or from whom the product to delete) via the `index` passed in.
+### Delete feature
 
-<div markdown="span" class="alert alert-info">:information_source:
-**Note:** If the index passed in is larger than the size of the list, an error will be raised and the deletion will terminate.
-</div>
+The `delete` feature will be elaborated in this section by its' functionality, the path execution with the aid of a sequence and an activity diagram.
+The details of __DeleteCommand__'s class implementation and its interactions with associated objects will also be discussed.
 
-Step 4. If the user wants to delete an entire warehouse/supplier entry, `model#deleteWarehouse` will remove the entry
-from the list in the `model`.
+#### What Delete Feature does
 
-If the user wants to delete a product inside the entry, the set of `Product` for the warehouse/supplier entry will be retrieved first.
-The `warehouse#getProductByName` or `supplier#getProductByName` will give the target product to delete from the name argument,
-and the retrieved product will be removed from the current product set in the selected warehouse/supplier entry.
-Afterwards, the updated product set will replace the existing set in this warehouse/supplier. The selected warehouse/supplier entry in the model is then 
-replaced by the updated warehouse/supplier with the target product deleted.
+The `delete` feature allows user to remove a warehouse or supplier __(case 1)__.
+<br>
+The `delete` feature also allows user to remove a product from a certain warehouse or supplier __(case 2)__.
+<br>
+<br>
+In case 1, `TYPE` needs to be set to `w` or `s`.
+<br>
+In case 2, `TYPE` needs to be set to `pw` or `ps`, and a `PRODUCT_NAME` needs to be specified.
+<br>
+<br>
+The deletion is limited to the items shown in the list displayed in GUI, and is done one item at a time.
+
+#### Path Execution of Delete Command
+
+The workflow of an `Delete` Command when it is executed by a user is shown in the activity diagram below:
```

Comment:
I think this should be a `Delete` command

Reply:
This is already fixed, thank!